### DISCUSSION TITLE
Can't get pan to work

### DISCUSSION THREAD

**Discussion by johnwc** [chartjs/chartjs-plugin-zoom#640] [Q&A]
I have a series of time data of 30 days that we are pulling back and showing on the chart. We are trying to get the zoom/pan plugin to work so that of the 30 days of data, the chart shows a portion and a user can pan forward and backward to scroll through the 30 days. In our case we would like to show 7 days of data, aka zoomed to the last  days. And then click to scroll previous days.

https://jsfiddle.net/1mx4n5kv/ What I have come up with so far, it selects the zoom time frame though instead of pan forward/backward.

**Comment by LeeLenaleee** [comment]
That's because of the drag true in your zoom, if you want to pan now you will need to hold the left ctr key.

If you want to pan with only a mouse click you will need to disable the zoom because it won't do good with plain pan together and remove the key modifier in the pan settings

**Comment by johnwc** [comment]
Can you update the jsfiddle and give me a working example. We actually don't want any zooming functionality. We just want to load 30/60/90 days worth of points into the chart, and the chart show the last(current) week. Allowing a user to drag/pan the chart to view previous days.
